src/main/java/hu/blackbelt/judo/generator/commons/GitIgnoreSynchronizer.java

```diff
@@ -113,10 +113,13 @@ public void addGeneratedFiles(Collection<GeneratorFileEntry> entries) {
                 }
             }
 
+            var alreadyExistedGlobPatterns = new ArrayList<>(newGitignoreLines);
```

Comment:
renaming `alreadyExistedGlobPatterns` to `presentGlobPatterns` would make it shorter and more accurate by definition.